### PR TITLE
Added list showing for tests

### PR DESCRIPTION
**Pull request by VolodymyrChykharivskyi**

### CODE REVIEW

File: src/app/admin/tests/list/test-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  @ViewChild(MatPaginator, { static: true }) paginator: MatPaginator;
+
+  constructor(protected httpService: HttpService, public dialog: MatDialog) {}
+
+  ngOnInit() {
+    this._loadSubjects();
+    this.viewAllTests();
+  }
+
+  _loadSubjects() {
```
Comment (AndriyIf) [inline src/app/admin/tests/list/test-list.component.ts]: We can use `private` access control modifier here

File: src/app/admin/admin.component.html [hunk 2]
```diff
@@ -9,6 +9,9 @@
          <mat-list-item >
             <a matLine routerLink="group">Groups</a>
          </mat-list-item>
+          <mat-list-item >
```
Comment (AndriyIf) [inline src/app/admin/admin.component.html]: check indentation

File: src/app/admin/tests/list/test-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    this.httpService.getRecords('subject').subscribe((result: Subject[]) => {
+      this.listSubjects = result;
+    });
+  }
+
+  getSubjectNameById(subjectId: number): string {
+    const subject = this.listSubjects.filter(subjectItem => {
```
Comment (AndriyIf) [inline src/app/admin/tests/list/test-list.component.ts]: Can we use `find` here?

File: src/app/admin/admin.component.html (hunk 2 above)
Comment (VolodymyrChykharivskyi) [reply src/app/admin/admin.component.html]: Done

File: src/app/admin/tests/list/test-list.component.ts (hunk 3 above)
Comment (VolodymyrChykharivskyi) [reply src/app/admin/tests/list/test-list.component.ts]: Done

**Review by AndriyIf — APPROVED** [review]

**Review by tatkaif — APPROVED** [review]

**Review by tatkaif — APPROVED** [review]

**Pull request merged**